docs/docker/ocpdiag.docker

```diff
@@ -0,0 +1,38 @@
+FROM centos:latest
```

Comment:
does it have to be centos? can this run on a smaller distro like alpine?

Reply:
I chose centos out of personal convenience, it doesn't have to be

Reply:
i should mention that this kinda depends on the usage; for demos, id say use alpine to make it smaller for distribution. for dev envs, centos can be fine because you'd normally build the container locally